docs/fsharp/using-fsharp-on-azure/table-storage.md

```diff
@@ -181,10 +198,12 @@ You can delete a table from a storage account. A table that has been deleted wil
 ## Next steps
 
 Now that you've learned the basics of Table storage, follow these links
-to learn about more complex storage tasks:
+to learn about more complex storage tasks and the Azure Cosmos DB Table API.
 
-- [Azure Storage APIs for .NET](/dotnet/api/overview/azure/storage)
-- [Azure Storage Type Provider](https://fsprojects.github.io/AzureStorageTypeProvider/)
-- [Azure Storage Team Blog](https://blogs.msdn.microsoft.com/b/windowsazurestorage/)
-- [Configure Azure Storage connection strings](/azure/storage/common/storage-configure-connection-string)
+- [Introduction to Azure Cosmos DB Table API](https://docs.microsoft.com/azure/cosmos-db/table-introduction)
+- [Storage Client Library for .NET reference](http://go.microsoft.com/fwlink/?LinkID=390731&clcid=0x409)
```

Comment:
Please, replace fwlink with [the actual link](https://docs.microsoft.com/en-us/dotnet/api/overview/azure/storage?view=azure-dotnet) as the issue #3391 really against fwlinks :).

Reply:
Ah, OK, fixed.